### PR TITLE
task2: automate app deploy to s3 with cloudfront

### PR DESCRIPTION
**Pull request by aliakseikrauchanka**
Manual:
S3: http://shop-app-first.s3-website-eu-west-1.amazonaws.com
CF: d21da4xlee91am.cloudfront.net

Auto:
S3: http://shop-app-first.s3-website-eu-west-1.amazonaws.com
CF: dt8un8mod3wbx.cloudfront.net

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  profile: dev
+  
+
+plugins:
+  - serverless-finch
+
```
Comment (VarvaraZadnepriak) [inline serverless.yml:31]: I think we should also include serverless-single-page-app-plugin here

Comment (aliakseikrauchanka) [reply serverless.yml:31]: Good catch! Seems it was the reason why my script showed some errors and did not finish the work till the end properly

But for some reason CloudFront has created distribution by even with issues in place ...

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# package directories
+jspm_packages
+
+# Serverless directories
+.serverless
```
Comment (sergeybolzan) [inline .gitignore:7]: the line already exists in the file

Comment (aliakseikrauchanka) [reply .gitignore:7]: Nice! Removed

**Pull request merged**